favicon.ico

```diff
@@ -0,0 +1 @@
+
```

Comment:
why?

Reply:
because I was getting an error. Isn't this what we did last time?